doc/services.md

```diff
@@ -0,0 +1,43 @@
+## Deployed CI services
+
+### ocurrent/ocurrent-deployer
+- `Dockerfile` on arches: x86_64
+  - branch `live-ci3` at `ocurrent/ci.ocamllabs.io-deployer:live-ci3`
+
+### ocurrent/ocaml-ci
+- `Dockerfile` on arches: x86_64, arm64
+  - branch `live-engine` at `ocurrent/ocaml-ci-service:live`
```

Comment:
To give an example of the linking generation I'm suggesting, this could end up looking like:

```suggestion
### [ocurrent/ocaml-ci](https://github.com/ocurrent/ocaml-ci)
- `Dockerfile` on arches: x86_64, arm64
  - branch `[live-engine](https://github.com/ocurrent/ocaml-ci/tree/live-engine)` at <https://ocaml.ci.dev/>
```

WDYT?

And, should we also include the `ocurrent/ocaml-ci-service:live`? I'm not exactly sure what additional info this gives us, but seems like it could be useful?